syntax = "proto3";

package limdongjin.ignasr.protos;

message AnalysisRequest {
  string reqId = 1;
  string userId = 2;
  string url = 3;
}
